### DISCUSSION TITLE
'module' is not defined complain on .eslintrc.cjs if you just follow getting started guide

### DISCUSSION THREAD

**Discussion by gavvvr** [typescript-eslint/typescript-eslint#6959] [Community Feedback]
Hi

I just follow the [Getting Started guide](https://typescript-eslint.io/getting-started/) and in first 3 steps I get an error which I am not sure how to deal with.

I do the following first simple steps:
- create an empty dir
- install deps with (which also creates `package(-lock).json`):
```bash
npm install --save-dev @typescript-eslint/parser @typescript-eslint/eslint-plugin eslint typescript
```
- and then I add `.eslintrc.cjs` or `.eslintrc.js` with the following content:
```javascript
module.exports = {
  extends: ['eslint:recommended', 'plugin:@typescript-eslint/recommended'],
  parser: '@typescript-eslint/parser',
  plugins: ['@typescript-eslint'],
  root: true,
};
```

Right after that [as a Step3](https://typescript-eslint.io/getting-started/#step-3-running-eslint) the guide suggests to execute `npx eslint .`, and I get the following error:

‼️ OOPS, I accidentally pressed CMD+Enter, the post is WORK-IN-PROGRESS

**Comment by bradzacher** [comment]
As per our contributing guidelines - please don't use discussions for bug reports or support requests.
https://typescript-eslint.io/contributing/discussions

This was also fixed last week - https://github.com/typescript-eslint/typescript-eslint/pull/6918

Duplicate of https://github.com/typescript-eslint/typescript-eslint/issues/6825
